### ISSUE TITLE
Getting 400 Bad request when using the swagger-ui.

### ISSUE THREAD

**Issue by sque** [noirbizarre/flask-restplus#15]
Using swagger-ui I always get `400 Bad request` if there is at least one argument for parsing. For endpoints that have no parameters it works just fine. Notice that the API works with no problem if I consume it from any other client, even the same browser by just requesting the URL in a different window.

After investigation I found that it was a problem of swagger-ui client that it always puts the 'Content-type: application/json` even for GET requests. Werkzeug or Flask(not sure) finds illegal putting content-type header without having any HTTP payload (the case of GET), thus the 400 Bad request.

**Comment by noirbizarre** [comment]
It's also flask-restful bug that has been fixed upstream but not yet released: flask-restful/flask-restful#366

**Comment by sque** [comment]
In the beginning I thought that putting Content-Type for GET request is violation of HTTP/1.1, so for me, it was definitely client's problem. Based on this [discussion](http://stackoverflow.com/questions/5661596/do-i-need-a-content-type-for-http-get-requests?answertab=votes#tab-top), I understand that there is no explicit rule in the HTTP/1.1 specification that forbids this rule. So, it was at flask-restful the right place to fix it, but it is also good that the swagger-ui client reduced this abuse of 'content-type' header with their new release.

Thanks, everyday learning :) Hope to see these changes released soon, it is a PITA bug for development :P

**Comment by noirbizarre** [comment]
Yeah, that's what's interesting in programming: you learn every day !! :)

I'm waiting too for flask-restplus release because me too I've been spending a lot of time trying to figure out what's happening !! :)
In the mean time, you could simply do until the new release:
```console
pip install http://github.com/flask-restful/flask-restful/tarball/231da4b
```

**Comment by noirbizarre** [comment]
It should have been fixed in Flask-restful 0.3.2.
